pkgs/applications/misc/waycorner/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, makeWrapper
+, rustPlatform
+, pkg-config
+, cargo
+, rustc
+, fetchFromGitHub
+, wayland
+, ...
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "waycorner";
+  version = "0.2.1";
+  src = fetchFromGitHub {
+    owner = "AndreasBackx";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-xvmvtn6dMqt8kUwvn5d5Nl1V84kz1eWa9BSIN/ONkSQ=";
+  };
+  cargoSha256 = "sha256-Dl+GhJywWhaC4QMS70klazPsFipGVRW+6jrXH2XsEAI=";
+  buildInputs = [
+    rustc
+    cargo
```

Comment:
It seems that we don't need these dependencies since this derivation uses `buildRustPackage`.